.github/workflows/publish.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions: write-all
+
+jobs:
+  create_release:
+    runs-on: windows-latest
+
+    steps:
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: true
+        prerelease: false
+
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+
+  build:
+    needs: create_release
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.1
+
+    - name: Restore
+      run: msbuild pbench.sln /t:restore /p:Configuration=Release /p:Platform=x86
+
+    - name: Build
+      run: msbuild pbench.sln /t:build /p:Configuration=Release /p:Platform=x86 /p:RestorePackages=false
+
+    - name: Upload pbench.msi to Release
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ needs.create_release.outputs.upload_url }}
+        asset_path: .\_bin\Release-x86\en-US\pbench.msi
+        asset_name: pbench.msi
```

Comment:
Is it possible to include tag name to package name? Something like pbench-0.4.0.msi would be great.

Reply:
Yes, it is possible

Reply:
Could you please add it then? Thanks in advance!